extensions/wikia/AdEngine/tests/AdEngine2ContextServiceTest.php

```diff
@@ -263,6 +267,19 @@ public function testGetContext(
 			$expectedTargeting['pageIsArticle'] = true;
 		}
 
+		if ( isset( $expectedTargeting['hasPoratableInfobox'] ) && $expectedTargeting['hasPoratableInfobox'] === true ) {
+			$wikiaMock = $this->getMockBuilder( 'Wikia' )
+				->disableOriginalConstructor()
+				->setMethods( [ 'getProps' ] )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "getProps" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)